ipn/prefs.go

```diff
@@ -681,3 +681,20 @@ func SavePrefs(filename string, p *Prefs) {
 		log.Printf("SavePrefs: %v\n", err)
 	}
 }
+
+// ProfileID is a system-wide unique identifier for a login profile.
```

Comment:
what's it look like? examples? restrictions on allowed chars?

Reply:
its auto-generated, added docs